docs/designs/conceptual-localization.md

```diff
@@ -77,22 +77,22 @@ Below kinds of mappings are considered to be supported and there is a **strong c
 
     Here is an string convention for loc repo name:
```

Comment:
In the `Repository` convention, we only need `{source-repo-name} -> {source-repo-name}.{locale}`. Repos with `{source-repo-name}.{source-locale} -> {source-repo-name}.{loc-locale}` convention will be renamed.

Reply:
yes, mooncake repos will be renamed as well.

Reply:
sure, will remove this convention